lib/3scale/dotenv.rb

```diff
@@ -1,18 +1,17 @@
 # Ensure the proper .env file is loaded
 #
-# Don't load any .env file for production
+# Don't load any .env file for production or staging
 # First, try to load `.env.#{ENV['RACK_ENV']}`
 # If doesn't exist, try to load .env.test
-# If doesn't exist, try to load .env
 def env_file
   file = ".env.#{ENV['RACK_ENV']}"
   return file if File.exists?(file)
 
-  File.exists?('.env.test') ? '.env.test' : '.env'
+  '.env.test'
```

Comment:
then maybe remove this line and replace the previous one with 
```
File.exists?(file) ? file : '.env.test'
```

?

Reply:
Done at https://github.com/3scale/apisonator/pull/375/commits/705a71893cf9ba2d862b1d6fbfc58913f6fad807